### PR TITLE
Bring NFT Card into style alignment with holaplex.com

### PR DESCRIPTION
**Pull request by echohtp**

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -241,38 +243,56 @@ const Home: NextPage<HomePageProps> = ({ storefront }) => {
...
-                        <div>
-                          <p className="text-right">55 {solSymbol}</p>
-                          <p className="text-right">Buy Now</p>
+                        <div className='p-4 border border-t-0 border-b-0 border-[#262626]'>
+                          <p className='text-lg'>{n.name}</p>
+                          <p className='text-sm text-[#a8a8a8]'>
+                            <span>
+                              {' '}
+                              <img
+                                src='https://arweave.cache.holaplex.com/jCOsXoir5WC8dcxzM-e53XSOL8mAvO0DetErDLSbMRg'
```
Comment (kespinola) [inline src/pages/index.tsx:269]: storefront logo is available in the database. If not exposed through graphql can you look into that?

File: src/pages/index.tsx [hunk 2]
```diff
@@ -241,38 +243,56 @@ const Home: NextPage<HomePageProps> = ({ storefront }) => {
...
-                        className="object-fill h-56 pb-2 rounded-lg"
-                      />
-                      <div className="grid grid-cols-2 gap-2">
-                        <div>
-                          <p>{n.name}</p>
+                <div key={n.address} className='h-[450px]'>
+                  <Link href={`/nfts/${n.address}`}>
+                    <div className='h-full p-4 m-0 transition duration-500 transform hover:scale-[1.02]' >
+                      <a>
+                        <div className='w-full border border-b-0 border-[#262626] h-[250px] overflow-clip'>
```
Comment (kespinola) [inline src/pages/index.tsx:256]: For the other cards we use a css trick to get the div to be a perfect square using after selector and margin. Once the box is in place and always based on the width. The image set with width height 100% and image-fit cover to position the center center cropped.

File: src/pages/index.tsx (hunk 1 above)
Comment (echohtp) [reply src/pages/index.tsx]: Yep, will do!

**Comment by echohtp** [comment]
Mobile 1 Column - To make mobile look right I had to hide the left filter bar
localhost_8080_mobile

2 Col 
localhost_8080_ 2-col

3 Col 
localhost_8080_3col

4 Col
localhost_8080_ 4col

5 Col 
localhost_8080_5col

**Pull request closed without merging**